### PR TITLE
Replace old shady pie with new MPU image

### PR DESCRIPTION
**Pull request by i-hardy**
## What does this change?

Replaces the adblock fallback 'shady pie' with a new MPU image.

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## Screenshots

Screenshot 2020-10-01 at 13 00 23

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Comment by mxdvl** [comment]
Could we use an SVG here? 😉

**Review by mxdvl — APPROVED** [review]
This is time-sensitive, so let’s do SVG in a subsequent PR!

File: static/src/javascripts/projects/common/modules/commercial/adblock-ask.js [hunk 1]
```diff
@@ -12,20 +12,9 @@ const supportUrl = `${supportSubscribeDigitalURL()}?acquisitionData=%7B%22compon
...
-                your devices
-            </h2>
-        </div>
-        <a class="contributions__adblock-button" href="${supportUrl}">
-            <span class="component-button__content">Subscribe now</span>
-        </a>
-        <img src="https://media.guim.co.uk/b437f809d9fa4c72336ccbead1730b6bb0965239/0_0_432_503/432.png" class="contributions__adblock-image" alt="" />
-    </div>
+    <a href="${supportUrl}">
+        <img src="https://media.guim.co.uk/3edc720c6210f873c7e75df4c46bea75615401d8/0_0_300_250/300.jpg" alt="" />
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/commercial/adblock-ask.js:16]: could this be a png or gif?

Comment (mxdvl) [reply static/src/javascripts/projects/common/modules/commercial/adblock-ask.js:16]: Unfortunately, no. I discussed with @i-hardy, and this is the only source file available.

This PR needs to be merged very shortly, too.

Comment (i-hardy) [reply static/src/javascripts/projects/common/modules/commercial/adblock-ask.js:16]: Yes, I would love to make this a little shinier but the timeline is unfortunately pretty tight. I think we're going to be reviewing the shady pie in the near future so hopefully there will be a chance to either significantly improve on this or get rid entirely.

**Comment by paperboyo** [comment]
Great idea about SVG, @mxdvl! Let’s see. Current asset has no allowance for HiDPI (will look bad there), is a JPEG (not suitable for graphicky style, visible artifacts) and weighs 17,8KB.

Here is a 2x resolution PNG, optimised to weigh 10KB and crisp: https://uploads.guim.co.uk/2020/10/02/Digisubs_MPU_c1_my_opt.png
Here is an SVG with 1 decimal place precision; because I guess live fonts are a no-no, weighing 28KB: https://uploads.guim.co.uk/2020/10/02/Digisubs_MPU_c1_my_d1_opt.svg

I would probably go with a PNG, because of the weight even at full 2x resolution allowance, but your call.

File: static/src/javascripts/projects/common/modules/commercial/adblock-ask.js (hunk 1 above)
Comment (paperboyo) [reply static/src/javascripts/projects/common/modules/commercial/adblock-ask.js:16]: See below.

Comment (mxdvl) [inline static/src/javascripts/projects/common/modules/commercial/adblock-ask.js:16]:
```suggestion
        <img src="https://uploads.guim.co.uk/2020/10/02/Digisubs_MPU_c1_my_opt.png" alt="" />
```

**Review by mxdvl — COMMENTED** [review]
Following @paperboyo’s comment, I think the PNG version is worth using.

---
<details>
<summary><strong>The following on SVG is slightly outside the scope of this PR…</strong></summary>

When it comes to SVG, it’s definitely possible to use fonts already present on the page, if it is embedded directly—not via an `img` tag. The best would be to embed only the subset of characters used in the image, which is maybe what https://vecta.io/nano does?

It is also worth noting that SVG compresses much better via `gzip`, so serving the SVG might be the best option anyway:

```zsh
 28K	Digisubs_MPU_c1_my_d1_opt.svg
8.0K	Digisubs_MPU_c1_my_d1_opt.svg.gz # compression levels 2 to 9
 12K	Digisubs_MPU_c1_my_d1_opt.svg.gz # compression level 1
 12K	Digisubs_MPU_c1_my_opt.png
 12K	Digisubs_MPU_c1_my_opt.png.gz
```

And lastly, the SVG was using a PNG as a background, so there was an extra 2KB to squeeze out by replacing it with a `rect`. Expand for details.

### After
```xml
<rect fill="#2e5581" height="100%" width="100%"/>
```
### Before… scroll right!
```xml
<defs>
  <mask id="a" x="0" y="0" width="600" height="500" maskUnits="userSpaceOnUse">
  <image width="600" height="500" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAlgAAAH0CAYAAADhUFPUAAAACXBIWXMAAAsSAAALEgHS3X78AAAI6ElEQVR4Xu3WQQ0AIBDAMMC/50MCIdmzfU/A9szMAgAgc14BAAB/DBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDEDBYAQMxgAQDELkkiB+Tjw9nIAAAAAElFTkSuQmCC" />
  </mask>
</defs>
<g mask="url(#a)">
  <image width="600" height="500" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAlgAAAH0CAYAAADhUFPUAAAACXBIWXMAAAsSAAALEgHS3X78AAAI7UlEQVR4Xu3WQQ3AIADAwDEniMIRvpmEhaTPu3cFdMy1zwMAQOb9CwAAuGOwAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIGawAABiBgsAIPYBR4UF4ad9APkAAAAASUVORK5CYII=" />
</g>
```

</details>

File: static/src/javascripts/projects/common/modules/commercial/adblock-ask.js (hunk 1 above)
Comment (i-hardy) [reply static/src/javascripts/projects/common/modules/commercial/adblock-ask.js]: That's a huge improvement, thanks Max!

**Pull request merged**